crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -343,3 +353,48 @@ pub struct MembersResponse {
     /// Collection of ambiguity changes that room member events trigger.
     pub ambiguity_changes: AmbiguityChanges,
 }
+
+#[cfg(test)]
+mod test {
+    use ruma::{
+        event_id,
+        events::{
+            room::message::RoomMessageEventContent, AnyMessageEvent, AnySyncMessageEvent,
+            AnySyncRoomEvent, MessageEvent,
+        },
+        MilliSecondsSinceUnixEpoch, RoomId, UserId,
+    };
+
+    use super::{Raw, RoomEvent, SyncRoomEvent, Unsigned};
+
+    #[test]
+    fn room_event_to_sync_room_event() {
+        let content = RoomMessageEventContent::text_plain("foobar");
+
+        let event: AnyMessageEvent = MessageEvent {
+            content,
+            event_id: event_id!("$xxxxx:example.org").to_owned(),
+            room_id: <&RoomId>::try_from("!someroom:example.com").unwrap().to_owned(),
```

Comment:
Like `event_id!` above, it would make sense to use `room_id!` here (and `user_id!` below).

Reply:
Ah, right. I initially quickly looked at the `RoomId` documentation on how to construct one and then didn't revisit it during the clean up. I've changed it now, though.